web/src/components/TokenDistributionModel/index.js

```diff
@@ -3,62 +3,71 @@ import React from 'react'
 import { Col, Row, Divider, Button, Tooltip } from 'antd'
 
 import Timeline from './Timeline'
+import MediaQuery from 'react-responsive'
 
 import tokenDistributionInfographic from '../../images/TokenDistributionInfographics.jpg'
-import styles from './distributionModel.module.css'
+import tokenDistributionInfographicMobile from '../../images/TokenDistributionInfographicsMobile.jpg'
+import styles from './distributionModel.css'
 
 function TokenDistributionModel() {
 
     return (
-        <div className={styles.tokenDistributionWrapper}>
+        <div className="tokenDistributionWrapper">
             <Row justify="center" align="center" type="flex">
                 <Col xs={{ span: 24 }} sm={{ span: 24 }} md={{ span: 24 }} lg={{ span: 24 }}>
-                    <h2 className={styles.tokenDistributionTitle}>Token distribution model</h2>
+                    <h2 className="tokenDistributionTitle">Token distribution model</h2>
```

Comment:
Why you're dropping use of `{styles.tokenDistributionTitle}` for global css classes?